diff --git a/dist/index.esm.js b/dist/index.esm.js
index 5a32539261de8c1c43512aa5365d85e97418b226..bd8f7c41ff0ae00b3a29db9267f73d03cc8e35e2 100644
--- a/dist/index.esm.js
+++ b/dist/index.esm.js
@@ -107,9 +107,10 @@ const NuxtModule = function(moduleOptions = {}) {
           return next(new Error("Invalid username or password"));
         }
         try {
+          const clientSecret = (options.auth || {}).clientSecret || process.env.DRUXT_AUTH_CLIENT_SECRET;
           const postData = new URLSearchParams({
             client_id: (options.auth || {}).clientId || process.env.DRUXT_AUTH_CLIENT_ID,
-            client_secret: (options.auth || {}).clientSecret || process.env.DRUXT_AUTH_CLIENT_SECRET,
+            ...clientSecret ? { client_secret: clientSecret } : {},
             ...data
           }).toString();
           const response = await axios.post(this.options.auth.strategies["drupal-password"].endpoints.token, postData, {
diff --git a/dist/index.ssr.js b/dist/index.ssr.js
index bc73e9b37475eac14af918742f728911861c7b05..ed6b261bc734e8d983e8b55a4c6ed25cd864328b 100644
--- a/dist/index.ssr.js
+++ b/dist/index.ssr.js
@@ -114,9 +114,10 @@ const NuxtModule = function(moduleOptions = {}) {
           return next(new Error("Invalid username or password"));
         }
         try {
+          const clientSecret = (options.auth || {}).clientSecret || process.env.DRUXT_AUTH_CLIENT_SECRET;
           const postData = new URLSearchParams({
             client_id: (options.auth || {}).clientId || process.env.DRUXT_AUTH_CLIENT_ID,
-            client_secret: (options.auth || {}).clientSecret || process.env.DRUXT_AUTH_CLIENT_SECRET,
+            ...clientSecret ? { client_secret: clientSecret } : {},
             ...data
           }).toString();
           const response = await axios__default["default"].post(this.options.auth.strategies["drupal-password"].endpoints.token, postData, {
